schemas/iso19139/src/main/plugin/iso19139/present/csw/gmd-brief.xsl

```diff
@@ -36,6 +36,8 @@
   <xsl:template match="gmd:MD_Metadata|*[@gco:isoType='gmd:MD_Metadata']">
     <xsl:variable name="info" select="geonet:info"/>
     <xsl:copy>
+      <xsl:apply-templates select="@*"/>
```

Comment:
I am not against this, but, is this because the attributes can have also weird namespaces?